src/everest/detached/jobs/everserver.py

```diff
@@ -193,41 +197,51 @@ def _write_hostfile(host_file_path, host, port, cert, auth) -> None:
         f.write(json_string)
 
 
-def _configure_loggers(
-    detached_node_dir: str,
-    everest_logs_dir: str,
-    logging_level: int,
-) -> None:
-    configure_logger(
-        name="res",
-        file_path=os.path.join(detached_node_dir, "simulations.log"),
-        log_level=logging.INFO,
-    )
-
-    configure_logger(
-        name="everserver",
-        file_path=os.path.join(detached_node_dir, "endpoint.log"),
-        log_level=logging.INFO,
-    )
+def _configure_loggers(config: EverestConfig) -> None:
```

Comment:
Is it possible for this function to only get the required components from EverestConfig and not the entire object?